VRDR.Messaging/CodingResponseMessage.cs

```diff
@@ -311,6 +311,78 @@ public CauseOfDeathEntityAxisEntry(string deathCertificateText, string causeOfDe
         }
     }
 
+    /// <summary>
+    /// Helper class for building a List&lt;CauseOfDeathEntityAxis&gt; from a flat file. Groups codes
+    /// into one <c>CauseOfDeathEntityAxisEntry</c> per line and sorts codes by position.
+    /// </summary>
+    public class CauseOfDeathEntityAxisBuilder
+    {
+        private SortedDictionary<int, SortedDictionary<int, string>> codes;
+
+        /// <summary>
+        /// Construct a new empty instance.
+        /// </summary>
+        public CauseOfDeathEntityAxisBuilder()
+        {
+            codes = new SortedDictionary<int, SortedDictionary<int, string>>();
+        }
+
+        /// <summary>
+        /// Build a List&lt;CauseOfDeathEntityAxis&gt; from the currently contained set of codes.
+        /// </summary>
+        /// <returns>cause of death entity axis coding list</returns>
+        public List<CauseOfDeathEntityAxisEntry> ToCauseOfDeathEntityAxis()
+        {
+            var list = new List<CauseOfDeathEntityAxisEntry>();
+            foreach (KeyValuePair<int, SortedDictionary<int, string>> pair in codes)
+            {
+                var entry = new CauseOfDeathEntityAxisEntry("", pair.Key.ToString());
```

Comment:
For clarity, perhaps putting `pair.Key.ToString()` into a temporary `lineNumber` variable would be helpful to indicate what the data is? Not sure about what you'd call `pair.Value.Values` if it made sense to do the same for that...

Reply:
OK.